src/main/java/qna/domain/Question.java

```diff
@@ -36,6 +44,27 @@ public Question(Long id, String title, String contents) {
         this.contents = contents;
     }
 
+    public DeleteHistories delete(User loginUser, LocalDateTime localDateTime) {
+        return DeleteHistories.from(
+            Stream.of(Collections.singletonList(deleteQuestion(loginUser, localDateTime)),
+                    answers.delete(loginUser, localDateTime).getDeleteHistories())
+                .flatMap(Collection::stream)
+                .collect(Collectors.toList())
+        );
```

Comment:
Stream의 concat을 써보거나, DeleteHistories의 정적팩터리 메서드로 표현해볼 수도 있을 것 같아요. 참고만 해주세요 😄 
```suggestion
        List<DeleteHistory> deleteHistories = Stream.concat(
                Stream.of(deleteQuestion(loginUser, localDateTime)),
                answers.delete(loginUser, localDateTime).getDeleteHistories().stream())
                .collect(Collectors.toList());
        return DeleteHistories.from(deleteHistories);
```

```suggestion
        return DeleteHistories.of(deleteQuestion(loginUser, localDateTime), answers.delete(loginUser, localDateTime));
```